### PR TITLE
Custom email

### PR DESCRIPTION
**Pull request by ivan-unfolds**

### CODE REVIEW

**Comment by rjmk** [comment]
Type tests are breaking!

File: src/admin/containers/email.js [hunk 1]
```diff
@@ -4,31 +4,39 @@ const { connect } = require('react-redux')
...
   , send_newsletter_reminder: remind
+  , send_custom: custom
   , emails
+  , submit_custom_email
+  , custom_emails
   , ...list_props
   }
-) =>
-  <div className='main-container email'>
+) =>{
```
Comment (rjmk) [inline src/admin/containers/email.js]: Any reason we added the

``` js
{
  return (
```

?

File: src/admin/dumb_components/custom_email_form.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const React = require('react')
+const { map, objOf, zipWith, merge, compose } = require('ramda')
+
+export default class CustomEmail extends React.Component {
```
Comment (rjmk) [inline src/admin/dumb_components/custom_email_form.js]: I think let's try and keep `dump_components` as function components. We could put an `onSubmit` on the form and then we can drop references to `this` by looking on the event object

Comment (rjmk) [inline src/admin/dumb_components/custom_email_form.js]: Also we use `React.createClass` in this project so far, so may as well stick with that when we need it

**Comment by rjmk** [comment]
I think the textarea on the custom email could probably be made broader. I think we could also do with a bit of space between the textarea and the submit button

**Comment by jay-meister** [comment]
Agreed. Possibly we got a bit excited when we pull requested. Will do.

**Pull request merged**